tools/flang1/flang1exe/semfunc2.c

```diff
@@ -2940,6 +2961,36 @@ iface_intrinsic(int sptr)
     dtyper = DT_QUAD;
     argdtype = DT_QUAD;
     break;
+  case I_QCOS: /* qcos */
```

Comment:
shouldn't this be hidden behind #ifdef as in the I_QLOG case?

Reply:
This `case` is guarded by the `#ifdef` on line 2958, added previously by commit 46ec115be.